platforms/r3-corda/configuration/roles/create/certificates/nms/vars/main.yaml

```diff
@@ -1,4 +1,8 @@
 userpassword_networkmap: admin
 mongodbPassword_networkmap: newdbnm
 rootca: "{{playbook_dir}}/build/corda/cordarootca"
-nmsca: "{{playbook_dir}}/build/corda/cordanetworkmap"
\ No newline at end of file
+nmsca: "{{playbook_dir}}/build/corda/cordanetworkmap"
+mongorootca: "{{playbook_dir}}/build/corda/mongorootca"
+mongodbca: "{{playbook_dir}}/build/corda/mongodbca"
+mongoroot_cert_subj: /C=US/ST=California/L=San Francisco/O=My Company Ltd/OU=DBA/CN=mongoDB
```

Comment:
reuse certificates from organization level certs in network.yaml

Reply:
we are providing certificates for nms and doorman only in network.yaml and not for their mongodb's.

Reply:
They can use the same subject. Does not have to be different